### ISSUE TITLE
Xdebug/ phpdbg check should be later in the infection process

### ISSUE THREAD

**Issue by BackEndTea** [infection/infection#325]
| Question    | Answer
| ------------| ---------------
| Infection version|  dev/master
| Test Framework version | n/a
| PHP version | 7.1
| Platform    | Fedora
| Github Repo | n/a

Running `bin/infection --help`, i expect to see the help overview, instead i see this

```
 [ERROR] Neither phpdbg or xdebug has been found. One of those is required by Infection in order to generate coverage   
         data. Either:                                                                                                  
                                                                                                                        
         - Enable xdebug and run infection again                                                                        
         - Use phpdbg: phpdbg -qrr infection                                                                            
         - Use --coverage option with path to the existing coverage report                                              
         - Use --initial-tests-php-options option with `-d zend_extension=xdebug.so` and/or any extra php parameters  
```

The same goes for running `bin/infection configure`. I expect to get into the configure modues, but instead i'm greeted by this error message.

Current work-around is to add the `--coverage=xyz` flag, so i get through the initial xdebug check.

**Comment by sidz** [comment]
>.<

**Comment by sidz** [comment]
Just enable xdebug :rofl: 

I'll take care about it

**Comment by BackEndTea** [comment]
> Just enable xdebug

True, but if i'd normally run things with phpdbg i'd expect to be able to run `--help` etc without it.

**Comment by sidz** [comment]
@BackEndTea it was a joke

**Comment by sanmai** [comment]
Isn't that just got fixed by #330?

**Comment by borNfreee** [comment]
yes, thanks @sanmai
